package/MDAnalysis/coordinates/TRC.py

```diff
@@ -0,0 +1,339 @@
+# -*- Mode: python; tab-width: 4; indent-tabs-mode:nil; coding:utf-8 -*-
+# vim: tabstop=4 expandtab shiftwidth=4 softtabstop=4
+#
+# MDAnalysis --- https://www.mdanalysis.org
+# Copyright (c) 2006-2017 The MDAnalysis Development Team and contributors
+# (see the file AUTHORS for the full list of names)
+#
+# Released under the GNU Public Licence, v2 or any higher version
+#
+# Please cite your use of MDAnalysis in published work:
+#
+# R. J. Gowers, M. Linke, J. Barnoud, T. J. E. Reddy, M. N. Melo, S. L. Seyler,
+# D. L. Dotson, J. Domanski, S. Buchoux, I. M. Kenney, and O. Beckstein.
+# MDAnalysis: A Python package for the rapid analysis of molecular dynamics
+# simulations. In S. Benthall and S. Rostrup editors, Proceedings of the 15th
+# Python in Science Conference, pages 102-109, Austin, TX, 2016. SciPy.
+# doi: 10.25080/majora-629e541a-00e
+#
+# N. Michaud-Agrawal, E. J. Denning, T. B. Woolf, and O. Beckstein.
+# MDAnalysis: A Toolkit for the Analysis of Molecular Dynamics Simulations.
+# J. Comput. Chem. 32 (2011), 2319--2327, doi:10.1002/jcc.21787
+#
+
+
+"""GROMOS11 trajectory reader --- :mod:`MDAnalysis.coordinates.TRC`
+====================================================================
+
+Reads coordinates, timesteps and box-sizes from GROMOS11 TRC trajectories.
```

Comment:
I added a reference to the _GROMOS data structures and format_ documentation in the mda documentation of the reader. Moreover, I added more details to the mda documentation explaining the capabilities of the reader as well as information that some blocks are not supported.

As mentioned in more detail in my comment below, I believe that the gromos documentation is proper. Please have a look into Chapter 2 and 4 of the linked pdf for documentation of all block types and associated format specifiers for reading and writing, that can be found within the gromos trajectory. If there is some important information missing, please inform me so that I can add it right there for the next GROMOS release 1.6.